packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -71,7 +75,13 @@ export const useRunnerStyle = ({
     specListWidth,
     containerHeight,
     containerWidth,
-    windowWidth: computed(() => windowWidth.value - collapsedNavBarWidth),
+    windowWidth: computed(() => {
```

Comment:
I'm not sure it's good to call this `windowWidth` and not `adjustedWindowWidth` or something, but plan to revisit that with the rest of these calculations soon.